scripts/kodiak/pmm_avellaneda.py

```
@@ -0,0 +1,445 @@
+        connector = self.connectors[self.config.exchange]
+        budget_checker = connector.budget_checker
+        
+        # Debug: Check balance before adjustment
+        self.logger().info(f"Available balance (USDC): {connector.get_available_balance('USDC')}")
+        self.logger().info(f"Total balance (USDC): {connector.get_balance('USDC')}")
+        
+        # Debug: Populate collateral for first order to see what happens
+        if proposal:
+            test_order = proposal[0]
+            populated = budget_checker.populate_collateral_entries(test_order)
+            self.logger().info(f"After populate_collateral_entries:")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)